src/color.js

```diff
@@ -0,0 +1,461 @@
+"use strict";
+
+/**
+ * color module
+ * @module color
```

Comment:
This is basically just a rewrite of https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/Color.js in ES6 module syntax.